.changeset/smart-hotels-perform.md

```diff
@@ -0,0 +1,6 @@
+---
+'@twilio-paste/design-tokens': patch
+'@twilio-paste/core': patch
+---
+
+Fixed shadow-border, shadow-border-weak, shadow-border-weaker, shadow-border-strong values to match respective color-border-\* values in all themes.
```

Comment:
Can this have a full table of changes?

If a consumer of Design Tokens were to look at the changelog produced by this, they should be able to figure out how to remap own implementations without digging into the commit history.